src/Window.cpp

```diff
@@ -917,6 +917,7 @@ void Window::onResetButtonPressed() {
     m_resetButton->setEnabled(false);
     m_resetButton->setText("Waiting");
     m_wasReset = true;
+    disabledMovement = true;
```

Comment:
I don't think we should set `m_didCrash = true` here. Pressing the reset button simulates when the *physical mouse crashes* without the virtual mouse knowing about it. Getting a "crash" response from the simulator simulates when the virtual mouse *is aware* that it crashed. They're slightly different things, we should treat them as such and only set `m_didCrash = true` when the mouse does, in fact, crash.